core/metrics-core-service/src/main/java/org/hawkular/metrics/core/service/DataAccess.java

```diff
@@ -67,21 +67,22 @@
 
     Observable<Row> findAllMetricsInData();
 
-    Observable<Integer> insertGaugeDatas(Observable<Metric<Double>> gauges, Function<MetricId, Integer> ttlFunc);
+    Observable<Integer> insertGaugeDatas(Observable<Metric<Double>> gauges,
+            Function<MetricId<Double>, Integer> ttlFunc);
 
     Observable<Integer> insertGaugeData(Metric<Double> metric);
 
     Observable<Integer> insertGaugeData(Metric<Double> metric, int ttl);
 
-    Observable<Integer> insertStringDatas(Observable<Metric<String>> strings, Function<MetricId, Integer>
-            ttlFetcher, int maxSize);
+    Observable<Integer> insertStringDatas(Observable<Metric<String>> strings,
```

Comment:
Would it make sense to modify the checkstyle / intellij / whatever rule here? Feel like it's designed for ancient 1024x768 screens. We could rise it a little bit, no? To me, this declaration easily fits in 1 line